docs/CHANGELOG.rst

```diff
@@ -43,6 +43,7 @@ Added
 * Add docker-compose command for PostgreSQL
 * API endpoint for adding ``Samples`` to ``Collections``
 * HISAT2 aligner
+* Test for ``generate_samples`` django-management command
```

Comment:
Change "django-management" to "django-admin" to be consistent with the previous change log entry.